mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -895,9 +905,9 @@ func flattenReplicationSpecs(rSpecs []matlas.ReplicationSpec) []map[string]inter
 	specs := make([]map[string]interface{}, 0)
 	for _, rSpec := range rSpecs {
 		spec := map[string]interface{}{
-			"id":             rSpec.ID,
-			"num_shards":     rSpec.NumShards,
-			"zone_name":      rSpec.ZoneName,
+			"id":         rSpec.ID,
+			"num_shards": rSpec.NumShards,
+			// "zone_name":      rSpec.ZoneName,
```

Comment:
why is zone_name commented out?

Reply:
Good catch! because the Cluster doesn't return the zone name variable in the response instead of that it returns the name of the zone like an object, e.j. :
```
"replicationSpecs": [
                {
                    "id": "5e72a859c4fb8e098f5b2c11",
                    "numShards": 1,
                    "regionsConfig": {
                        "EU_CENTRAL_1": {  // <---- here
                            "analyticsNodes": 0,
                            "electableNodes": 3,
                            "priority": 7,
                            "readOnlyNodes": 0
                        }
                    },
                    "zoneName": "Zone 1"
                }
            ],
```
Then when a ran a cluster test I got an error in the log, so I catched this, so to avoid errors I commented to check this with @marinsalinas, so @marinsalinas do you know if the response was always like this?

Reply:
I'm.mot sure but you can remove it and then add validation to set if it's.presemt.

Reply:
I will check it to be ensured :)

Reply:
I just added a validation for that attribute, because it was missing